tools/packaging/packaging.psm1

```diff
@@ -1743,7 +1745,7 @@ function New-NuSpec {
     $nuspecTemplate = $packagingStrings.NuspecTemplate -f $PackageId,$PackageVersion
     $nuspecObj = [xml] $nuspecTemplate
 
-    if ( ($Dependency -ne $null) -and $Dependency.Count -gt 0 ) {
+    if ( ($null -ne $Dependency) -and $Dependency.Count -gt 0 ) {
```

Comment:
I wonder that we change this.

Reply:
Do you mean "**why** we change this"? It's reported as a warning from Script Analyzer.